modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -904,6 +926,11 @@ definition {
 		return ${relationshipName};
 	}
 
+	@summary = "Default summary"
+	macro deleteObjectDefinition(objectDefinitionName = null) {
+		ObjectDefinitionAPI._deleteObjectDefinition(objectDefinitionName = ${objectDefinitionName});
+	}
```

Comment:
FYI, maybe we can use 'ObjectAdmin.deleteObjectViaAPI(objectName = "Student");' to delete an object
cc @yanan

Reply:
I am ok to either use UI or API here. But if use API, then we can rename the _deleteObjectDefinition instead of create a new macro cc @JoyceWang08211

Reply:
Good idea, then we do not need to create a new macro, thanks @Hijky